src/python_testing/TC_BOOLCFG_4_2.py

```diff
@@ -0,0 +1,176 @@
+#
+#    Copyright (c) 2023 Project CHIP Authors
+#    All rights reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+#
+
+import logging
+
+import chip.clusters as Clusters
+from chip.interaction_model import InteractionModelError, Status
+from matter_testing_support import MatterBaseTest, TestStep, async_test_body, default_matter_test_main
+from mobly import asserts
+
+sensorTrigger = 0x0080_0000_0000_0000
+sensorUntrigger = 0x0080_0000_0000_0001
+
+
+class TC_BOOLCFG_4_2(MatterBaseTest):
+    async def read_boolcfg_attribute_expect_success(self, endpoint, attribute):
+        cluster = Clusters.Objects.BooleanStateConfiguration
```

Comment:
you could have defaulted the endpoint into your helper to shrink-ify the calls below

Reply:
We can not really do that, since it needs to be dependent on what endpoint of the device being tested have the cluster implemented. The EP is grabbed from the user params.